build/buildkite-build.sh

```diff
@@ -59,15 +59,24 @@ do
   sleep 1
 done
 
-# Set up the schema and run tests
+# Set up the schema generate enums and run tests
 echo "--- :scala: Running tests"
 /usr/local/bin/sbt                                        \
   -jvm-opts build/buildkite-jvmopts                       \
   -Docs3.skipDependencyUpdates                            \
   -Docs3.databaseUrl=jdbc:postgresql://$HOST_AND_PORT/gem \
   sql/flywayMigrate                                       \
+  genEnums                                                \
   test
 
+# Check git status. if genEnums generated something mismatching, this should fail
+echo "--- :git: check code generation is current"
+if [ -n "$(git status --untracked-files=no --porcelain)" ]; then
+  # Uncommitted changes in tracked files
+  echo "--- :shrug: Enum generation produced unmatching files"
```

Comment:
I'm not sure why we need this, but I guess it doesn't hurt anything.  Won't it skip new enums though since they will initially be untracked.

Is it necessarily only changes produced by `genEnums`?  It looks like you can't continue with the build until you commit things you've modified in general.

Reply:
right, we could remove the `untracked` check

Reply:
`if [ "$BUILDKITE" = "true" ]; then` ...